README.md

```diff
@@ -1,18 +1,92 @@
-# amplify
+# Amplify
 
-<a href="https://travis-ci.org/stkent/amplify"><img src="https://travis-ci.org/stkent/amplify.svg"></a> [ ![Download](https://api.bintray.com/packages/stkent/android-libraries/amplify/images/download.svg) ](https://bintray.com/stkent/android-libraries/amplify/_latestVersion) [![Coverage Status](https://coveralls.io/repos/stkent/amplify/badge.svg?branch=master&service=github)](https://coveralls.io/github/stkent/amplify?branch=master)
+Respectfully request feedback in your Android app.
 
-# Inline Licenses
+<!-- TODO: add example image or gif here -->
 
-You can automatically generate license headers in new source files using the Gradle command:
+# Project Status
 
-    ./gradlew licenseFormat
+<a href="https://travis-ci.org/stkent/amplify"><img src="https://travis-ci.org/stkent/amplify.svg"></a> [![Coverage Status](https://coveralls.io/repos/stkent/amplify/badge.svg?branch=master&service=github)](https://coveralls.io/github/stkent/amplify?branch=master) [![Download](https://api.bintray.com/packages/stkent/android-libraries/amplify/images/download.svg)](https://bintray.com/stkent/android-libraries/amplify/_latestVersion)
+
+# Getting Started
+
+In your `build.gradle`:
+
+    dependencies {
+        compile 'com.github.stkent:amplify:0.1.0'
+    }
+    
+In your `Application` class:
+
+    public class ExampleApplication extends Application {
+        @Override public void onCreate() {
+            super.onCreate();
+            AmplifyStateTracker.get(this)
+                    .configureWithDefaults()
+                    .setLogLevel(BuildConfig.DEBUG ? Logger.LogLevel.DEBUG : Logger.LogLevel.NONE);
+        }
+    }
+
+In your xml layout(s):
+
+    <com.github.stkent.amplify.views.AmplifyView
+        android:id="@+id/amplifyView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+In the corresponding `Activity`/`Fragment`/`View` class (typically presented immediately after a successfully-completed 'transaction'):
+
+    final AmplifyView amplifyView = (AmplifyView) findViewById(R.id.amplifyView);
+    
+    ...
+    
+    AmplifyStateTracker.get(this).promptIfReady(amplifyView);
+
+# Configuring
+
+## Default Rules
+
+## Default Prompt UI
+
+# Customizing
 
-# Tests
+## Custom Events
 
-You can execute library tests only using the Gradle command:
+## Custom Rules
 
-    ./gradlew :amplify:test
+## Custom Prompt UI
+
+# Contributing
+
+## Conventions
+
+Code style and correctness is also enforced by several checkers. Running the Gradle command
+
+    ./gradlew amplify:check
```

Comment:
We need to think what to do here because we have the fail on error set to false so Gnag can get a full report together but this also allows the check to complete even with error. I wonder if we should just add another task to Gnag allowing you to run reports without trying to send them to Github, what do you think?